frontend/src/components/common/RequiredAsterisk.tsx

```diff
@@ -0,0 +1,12 @@
+import { Text } from "@chakra-ui/react";
+import React from "react";
```

Comment:
nit: I think we've been using React as first import

Reply:
Sounds good